src/bind.c

```diff
@@ -3,12 +3,12 @@
 #include "decl/bind-decl.h"
 
 // [[ register(external = TRUE) ]]
-r_obj* ffi_rbind(r_obj* ffi_call, r_obj* op, r_obj* args, r_obj* env) {
+r_obj* ffi_rbind(r_obj* ffi_call, r_obj* op, r_obj* args, r_obj* frame) {
   args = r_node_cdr(args);
 
-  struct r_lazy call = { .x = env, .env = r_null };
+  struct r_lazy call = { .x = syms_dot_call, .env = frame };
```

Comment:
Fairly confident that the environment we get from `.External2()` is the same one we'd get from calling `environment()` and passing that through